src/override/functions.js

```diff
@@ -0,0 +1,63 @@
+export const getResource = (resources, resourceName) =>
+  resources.find(({name}) => resourceName === name);
+
+export const getResourceField = (resource, resourceFieldName) =>
```

Comment:
should be `export function` right? (same for all functions in this file)

Reply:
Done. But just curious, why using `export function` instead of `export const` an arrow function ?